src/Behat/MinkExtension/Context/MinkContext.php

```diff
@@ -465,11 +465,11 @@ public function assertNumElements($num, $element)
     }
 
     /**
-     * Checks, that checkbox with specified in|name|label|value is checked
+     * Checks, that checkbox with specified id|name|label|value is checked
      * Example: Then the "remember_me" checkbox should be checked
      * Example: And the "remember_me" checkbox is checked
      *
-     * @Then /^the "(?P<checkbox>(?:[^"]|\\")*)" checkbox should be checked$/
+     * @Then /^the "(?P<checkbox>(?:[^"]|\\")*)" checkbox (?:is|should be) checked$/
```

Comment:
I don't understand why it breaks any translation.
It breaks auto-completion but it continues to be correct no ?